include/rmm/device_scalar.hpp

```diff
@@ -234,19 +244,23 @@ class device_scalar {
    * specified to the constructor, then appropriate dependencies must be inserted between the
    * streams (e.g. using `cudaStreamWaitEvent()` or `cudaStreamSynchronize()`), otherwise there may
    * be a race condition.
+   *
+   * @return Constant pointer to underlying device memory
```

Comment:
```suggestion
   * @return Const pointer to underlying device memory
```
?
For consistency with change above.